app/src/main/java/com/babylon/wallet/android/data/transaction/TransactionClient.kt

```diff
@@ -295,8 +291,8 @@ class TransactionClient @Inject constructor(
                     )
                 )
             }
-        } else {
-            return Result.failure(DappRequestException(DappRequestFailure.GetEpoch))
+        } ?: run {
```

Comment:
could just be `?: Result.failure(DappRequestException(DappRequestFailure.GetEpoch))`

Reply:
Yes true 👍